test/unit/resources/platform_test.rb

```diff
@@ -0,0 +1,99 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Richter
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::Os' do
```

Comment:
Looks like you're testing the OS resource, but the file is named `platform_test`?